src/app/common/TagList.vue

```diff
@@ -1,5 +1,11 @@
 <template>
-  <KTruncate>
+  <component
+    :is="props.shouldTruncate ? 'KTruncate' : 'div'"
```

Comment:
Should we use the `AnonymousComponent` here (which is kinda the `absent` of component-land)?

Its an alternative to `if/else` when you don't want anything in the `else` portion, i.e. you really want to use an `if`

Reply:
But we do need the `div` to attach `.tag-list` as without KTruncate, we don’t have tag list styles.

Reply:
Ah gotcha, cool lets stick with the `div` then